<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TankGame</class>
 <widget class="QWidget" name="TankGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>810</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>760</width>
     <height>760</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_t1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>780</y>
     <width>371</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_t2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>780</y>
     <width>371</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>graphicsView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
